src/etc/inc/services.inc

```diff
@@ -2204,6 +2204,50 @@ function services_unbound_configure($restart_dhcp = true) {
 			}
 		}
 
+		/* DNS Resolver python integration */
+		$base_folder = '/usr/local';
+		foreach (array('/bin', '/lib') as $dir) {
+			$validate = exec("/sbin/mount | /usr/bin/grep '{$g['unbound_chroot_path']}{$base_folder}{$dir}' 2>&1");
+			if (isset($config['unbound']['python']) && !empty($config['unbound']['python_script'])) {
+
+				// Add DNS Resolver python integration
+				if (empty($validate)) {
+					if (!is_dir("{$g['unbound_chroot_path']}{$base_folder}{$dir}")) {
+						exec("/bin/mkdir -p {$g['unbound_chroot_path']}{$base_folder}{$dir} 2>&1");
```

Comment:
Why not use `safe_mkdir("{$g['unbound_chroot_path']}{$base_folder}{$dir}");`?

Reply:
Is that equivalent to the "-p" arg?

Reply:
It has the same net effect as `-p`.

https://github.com/pfsense/pfsense/blob/master/src/etc/inc/util.inc#L2023